test/AElf.Contracts.SolidityContract.Tests/contracts/release_version.sol

```diff
@@ -0,0 +1,7 @@
+contract release {
+    function print_then_error(int8 num) public returns (int8) {
+        print("Hello!");
```

Comment:
just wondering is the "Hello" print intentional here? or was it for debugging earlier on? thx